src/application/agora/widgets/proposal-handle.container.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from 'react';
+
+import { useCommandQuery } from 'shared/messaging';
+
+import { getAgoraUsernameFromTwitterUsername } from '../utils';
+import { GetAgoraProposalsCommand } from '../commands';
+import { ProposalData } from '../types';
+
+import { Proposal } from './proposal';
+
+interface Properties {
+  handle: string;
+}
+
+export const ProposalHandleContainer = ({ handle }: Properties) => {
+  const [currentProposalIndex, setCurrentProposalIndex] = useState<number>(0);
```

Comment:
you don't need to pass generic here because typescript will infer the type based on provided default value